cgi-bin/LJ/Setting/UserMessaging.pm

```diff
@@ -32,7 +32,7 @@ sub option {
     my ($class, $u, $errs, $args) = @_;
     my $key = $class->pkgkey;
 
-    my $usermsg = $class->get_arg($args, "usermsg") || $u->prop("opt_usermsg");
+    my $usermsg = $class->get_arg($args, "usermsg") || $u->opt_usermsg);
```

Comment:
Oops! It looks like there is a stray close paren here that also needs to be removed.